rosdep/python.yaml

```diff
@@ -1325,6 +1325,10 @@ python-debian:
     yakkety_python3: [python3-debian]
     zesty: [python-debian]
     zesty_python3: [python3-debian]
+python-decorator:
+  debian: [python-decorator]
+  fedora: [python-decorator]
+  ubuntu: [python-decorator]
```

Comment:
@mgrrx any chance we add Gentoo and OSX to the mix?

Also, please provide references for each key to aid the review process, as required by our [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions).

Reply:
see in the comment below